.github/workflows/live-network-tests.yaml

```diff
@@ -0,0 +1,64 @@
+name: Live Network Tests
+
+on:
+  schedule:
+    - cron: '0 6,18 * * *'   # Runs daily at 6 AM and 6 PM UTC
+  workflow_dispatch: {}         # Allow manual triggering
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      LIVE_TESTS: "true"
+      LIVE_TEST_PREPROD_HOLESKY_KEY: ${{ secrets.LIVE_TEST_PREPROD_HOLESKY_KEY }}
+      LIVE_TEST_TESTNET_HOLESKY_KEY: ${{ secrets.LIVE_TEST_TESTNET_HOLESKY_KEY }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.24'
+
+      - name: Install dependencies
+        run: go mod download
+
+      - name: Run Live Network Tests
+        run: go test ./test/v2/live -v -timeout 60m
+
+      - name: Notify Slack
+        if: always()
+        run: |
+          if [ "${{ job.status }}" == "success" ]; then
+            COLOR="good"
+            STATUS_EMOJI="✅"
+            MENTION=""
+          else
+            COLOR="danger"
+            STATUS_EMOJI="❌"
+            MENTION="@here "
+          fi
+
+          PAYLOAD=$(jq -n \
+            --arg channel "#da-live-tests" \
+            --arg text "${MENTION}Live Network Tests completed, status: ${STATUS_EMOJI} ${{ job.status }}" \
+            --arg title "logs" \
+            --arg title_link "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}" \
+            --arg color "$COLOR" \
+            '{
+              channel: $channel,
+              text: $text,
+              attachments: [
+                {
+                  color: $color,
+                  title: $title,
+                  title_link: $title_link
+                }
+              ]
+            }')
+
+          curl -X POST -H "Authorization: Bearer ${{ secrets.DA_TEST_REPORTER_SLACK_OATH_TOKEN }}" \
```

Comment:
should this secret be named `LIVE_TEST_...` to have same prefix as the secret keys?